code/modules/surgery/organs/lungs.dm

```diff
@@ -243,6 +243,7 @@
 			H.Unconscious(60) // 60 gives them one second to wake up and run away a bit!
 			if(SA_pp > SA_sleep_min) // Enough to make us sleep as well
 				H.Sleeping(max(H.AmountSleeping() + 40, 200))
+				H.Anesthetized(max(H.AmountAnesthetized() + 40, 200)) //N2O is an anesthetic agent
```

Comment:
I feel like Anesthetized should conditionally apply the Sleeping effect

Reply:
isn't this way better as a trait or something? why does anesthetics even need a status effect?

Reply:
status effects are way heavier than traits and unless you want something where it slowly wears off after you stop using it it's a waste (and yes i can see you doing that here but uh if they're conscious they should probably feel pain tbh)